src/main/java/com/couchbase/lite/support/JsonDocument.java

```diff
@@ -36,7 +36,13 @@ public Object jsonObject() {
                 tmp = new LazyJsonArray<Object>(json);
             } else {
                 try {
-                    tmp = Manager.getObjectMapper().readValue(json, Object.class);
+                    // this is for Jackson 2.5.0
```

Comment:
json is byte[] which is from `Cursor.getBlob()`. And json byte array is ended with '\0'. '\0' causes parsing problem with Jackson 2.5.0. We did not observe this problem with Jackson 1.9.2.